README.md

```diff
@@ -2,7 +2,14 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+First,install dependencies
+
+```bash
+npm install
+# or
+yarn install
+
+Second, run the development server:
 
 ```bash
 npm run dev
```

Comment:
It would be nice if you could also remove the npm instructions that were present on the original README.